dev-utils/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - "cluster.routing.allocation.disk.threshold_enabled=false"
       - "discovery.type=single-node"
       - "path.data=/usr/share/elasticsearch/data/${STACK_VERSION}"
+      - "indices.id_field_data.enabled=true"
```

Comment:
seems like the old version does not support it, we would need to feature detect and only do it like how apm-integration-testing is doing it.